dandi/files/bases.py

```diff
@@ -453,6 +456,14 @@ def get_metadata(
         metadata.path = self.path
         return metadata
 
+    def _get_inspector_versions(self):
+        url = "https://pypi.org/pypi/nwbinspector/json"
+        data = json.loads(requests.get(url=url).text)
```

Comment:
```suggestion
        data = requests.get(url).raise_for_status().json()
```

Assuming you can't get etelemetry to work.

Reply:
Behavior is not similar - calling `raise_for_status` when response is a success returns `None`. Updated to use the direct `.json()` call, though.

Reply:
`etelemetry` working now anyhow